stack-component-versions.json

```diff
@@ -74,30 +82,30 @@
       "url": "https://github.com/mlflow/mlflow",
       "ourLatest": "2.8.0"
     },
-    {
-      "name": "Airflow",
-      "url": "https://github.com/apache/airflow"
-    },
     {
       "name": "ActivePieces",
-      "url": "https://github.com/activepieces/activepieces"
+      "url": "https://github.com/activepieces/activepieces",
+      "ourLatest": "0.24.1"
     },
     {
       "name": "Bytebase",
-      "url": "https://github.com/bytebase/bytebase"
+      "url": "https://github.com/bytebase/bytebase",
+      "ourLatest": "2.16.0"
     },
     {
       "name": "ClickHouse",
-      "url": "https://github.com/ClickHouse/ClickHouse"
+      "url": "https://github.com/ClickHouse/ClickHouse",
+      "ourLatest": "23.7.4"
     },
     {
       "name": "Datahub",
-      "url": "https://github.com/thedatahub/Datahub",
-      "ourLatest": "release"
+      "url": "https://github.com/datahub-project/datahub",
+      "ourLatest": "0.10.5"
```

Comment:
I remember this being an issue with DataHub -- can you also check the message on MM to make sure it's correct?